onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -7698,6 +7698,19 @@ def aten_sinh(self: TFloat) -> TFloat:
     return op.Sinh(self)
 
 
+@torch_op(("aten::slice.Tensor"), trace_only=True, complex=True)
+def aten_slice_complex(
+    self: TTensor,
+    dim: int = 0,
+    start: Optional[INT64] = None,
+    end: Optional[INT64] = None,
+    step: Optional[INT64] = None,
+) -> TTensor:
+    """slice.Tensor(Tensor(a) self, int dim=0, SymInt? start=None, SymInt? end=None, SymInt step=1) -> Tensor(a)"""
+
+    return aten_slice(self, dim, start, end, step)
```

Comment:
Should we turn negative axes into positive? I don't remember how complex are represented, real with one more dimension or...

Reply:
That’s a good point

Reply:
Done